src/components/Modals/Send/views/Status.tsx

```diff
@@ -167,8 +167,9 @@ export const Status: React.FC = () => {
       address: fromAccountId(accountId).account,
       chainId: fromAccountId(accountId).chainId,
       maybeSafeTx,
+      txStatus,
```

Comment:
That one does look sane!

Reply:
I removed them, as they are using the default explorer anyway it's a non issue in this scope, focusing on swap only!